code/modules/paperwork/paper.dm

```diff
@@ -40,10 +40,14 @@
 
 /obj/item/weapon/paper/update_icon()
 	if(burn_state == ON_FIRE)
+		// if(!istype(src, /obj/item/weapon/paper/paperplane)) //placeholder for paperplane_fire code
 		icon_state = "paper_onfire"
 		return
 	if(info)
-		icon_state = "paper_words"
+		if(!istype(src, /obj/item/weapon/paper/paperplane))  //don't update paperplanes
```

Comment:
No really, listen to @xxalpha, don't bother with the if stuff at the paper level. We call that snowflake, and we don't like it.